guides/mapping-and-blending/avoiding-css-conflicts/README.md

```diff
@@ -113,11 +94,15 @@ We recommend the following rules when using BEM selectors in Starcounter apps.
 }
 ```
 
+* __Never nest blocks inside blocks and elements inside elements.__
+
+ If there’s need for more nesting, it means there’s too much complexity and the elements should be stripped down into smaller blocks.
+
```

Comment:
What do you mean?

Is the following code valid?

```html
<button class="app-menu__button">
    <span class="app-menu__description">Description</span>
    <i class="app-menu__icon fa fa-circle"></i>
</button>
```

Would be possible to get an example of what's not recommended here?